### ISSUE TITLE
overriding notification email subject

### ISSUE THREAD

**Issue by RyanQuey** [simukappu/activity_notification#33]
I am trying to override the subject for the emails that get sent out.

I put the following in my notifiable model ("app/models/post.rb", where I have `acts_as_notifiable` set):

```
def overriding_notification_email_subject(target, key)
  if key == "weekly.unopened"
    "You have unread notifications"
  else
    if key == "new"
      "New Post"
    elsif key == "update"
      "A Post has been updated"
    elsif key == "destroy"
      "A Post has been destroyed"
    end
  end
end
```

However, the subject isn't getting changed.

I also confirmed in rails console that the following returns `true`: 

```
n = notification
n.notifiable.respond_to?(:overriding_notification_email_subject) && 
  n.notifiable.overriding_notification_email_subject(n.target, n.key).present?
```

(I'm trying to follow the logic of activity_notification/lib/activity_notification/mailers/helpers.rb:61-63).

Please let me know what I need to change to fix the subject headers, thanks!

**Comment by simukappu** [comment]
Hi,

This overriding function is a new and is not released as latest version yet.
https://github.com/simukappu/activity_notification/pull/27
We will release this as a next version one or a few weeks later.
If you want use this function now, please try gem in master branch from of this git repository.
Thanks!

**Comment by RyanQuey** [comment]
Perfect, thanks for the quick reply!

I'm glad you guys added it, very helpful feature!

**Comment by simukappu** [comment]
Just released the latest 1.4.2 version.
https://rubygems.org/gems/activity_notification/versions/1.4.2
Please try this.
Many thanks!

**Comment by RyanQuey** [comment]
This is wonderful, thank you!!
